decentraland/components/presentational/MyAccountCreationStatusItem.js

```diff
@@ -18,13 +18,15 @@ export default function MyAccountCreationStatusItem({ name, status }) {
   );
 }
 
+const ICON_SIZE = 30;
+const FONT_SIZE = (ICON_SIZE * 7) / 10;
+
 function renderIcon(status) {
-  const ICON_SIZE = 20;
   if (status === ActionStatus.DONE) {
     return (
       <Ionicons
```

Comment:
Make sense to use icons from [`native-base`](https://docs.nativebase.io/Components.html#icon-def-headref)?

Reply:
No strong preference. Happy to make that change. Will the icons look more native on each platform? Are they guaranteed to have a fixed width unlike these icons?

Reply:
Seems that `native-base` `Icon` component uses `Ionicicons` by default. About size:

> In case if you want to include icon with custom color, size etc then that should go into style.

Reply:
Ah ok. Then maybe I’ll stick with the Expo-bundled Ionicons since we’re more likely to move away from NativeBase some day than we are from Expo. Noted about styling, thanks

Reply:
Resolving this